src/main/kotlin/net/ccbluex/liquidbounce/utils/client/FogExtension.kt

```diff
@@ -22,3 +14,11 @@ fun Fog.copy(
         shape, color.r,
         color.g, color.b, color.a)
 }
+
+// could be merged into `Color4b` as generics, but not sure if izuna would be fine with that.
+data class FloatColor(val r: Float, val g: Float, val b: Float, val a: Float) {
```

Comment:
This can replaced by /255f when constructing the Fog object.

Reply:
but we would lose some precision, not sure if anyone cares :skull: